modules/ROOT/pages/design-publish.adoc

```diff
@@ -1,59 +1,63 @@
-<2> *API version*: The version number of the API specification. For a description of this field, see the "Filling in the *API version* Field" section.
-<3> *Business group ID*: The ID of the business unit in Anypoint Platform to which the project belongs. You should keep the value as it is, except in unusual cases.
-<4> *Asset name*: The name that Anypoint Exchange uses to identify the published project. You should keep the value as is, except in unusual cases.
-<5> *Asset ID*: The ID that Anypoint Exchange uses to identify the published project. You should keep the value as is, except in unusual cases.
-<6> *Add to API Manager*: Specifies that you want to publish the API specification to an API portal. When you click *Publish to Exchange*, API Designer also publishes the specification to API Manager, where the specification is listed in the *API Administration* section.
 . After completing the required fields, click *Publish to Exchange*.
 +
-If your project includes files that are not referenced by the root file either directly or indirectly, a dialog asks you to confirm that you do want to publish the project together with those unreferenced files. By default, API Designer excludes them from the published project in Anypoint Exchange.
+If your project includes files that are not referenced by the root file either directly or indirectly, a dialog asks you to confirm that you  want to publish the project with the unreferenced files. By default, API Designer excludes them from the published project in Anypoint Exchange.
+
+NOTE: If you are using the non-crowd version of Anypoint Platform and want to add the asset to API Manager while publishing to Exchange, select *Add to API Manager*. This functionality is not available in the crowd version and instead you can link API instances in API Manager to assets in Exchange.
+
+[[asset-version]]
+== Asset Version
+
+Anypoint Exchange requires an asset version number for each API specification published. The asset version must be specified using semantic versioning rules. For details, see xref:exchange::to-change-raml-version[Change the Version of an Asset].
+
+The content of the *Asset version* field on the publish dialog depends on the following:
+
+* If you are publishing the specification for the first time, you can use any valid number based on the semantic versioning scheme. A typical starting version is `1.0.0`. 
+* If you are republishing the specification, API Designer populates the *Asset version* field with the next patch level number based on the number used when the specification was last published. You can change the number, but the republish will fail if you use a version number that has already been used to publish a stable version of this API.
+* If the specification is being republished and you select the `In Development` state, you can can use the same number as a previously-published in-development version of the API, but not the same number as a previously-published stable version.
```

Comment:
Parallelism:
```suggestion
* If you are republishing the specification and select `In Development`, you can use the same number as a previously published in-development version of the API, but not the same number as a previously published stable version.
```

Reply:
Do we need to make clear that there are two types of asset version numbers: those for in-dev versions and those for stable versions?